.github/actions/rng/action.yml

```diff
@@ -0,0 +1,25 @@
+name: "RNG"
+description: "Generate random bytes using /dev/urandom. WARNING: only use for non-cryptographic purposes (the results will show in logs)."
+inputs:
+  length:
+    description: "Length to generate (in bytes)."
+    required: true
```

Comment:
nit: Is there a reason it shouldn't have a default?

Reply:
I'm not sure we can decide a default length. I could have it be 64 bytes default... which value would you set as default?

Reply:
I've made the default 16, which is 128bit is good for most purposes